### DISCUSSION TITLE
New version ?

### DISCUSSION THREAD

**Discussion by DzzD** [supermerill/SuperSlicer#2797] [Q&A] [answered]
Hello,

I was wondering if a new release is planned soon ? 
also do you plan to merge from Prusa or is it too different now ?

**Comment by neophyl** [comment, marked as answer]
Define “new”. The latest nightly for windows was 6 days ago. It’s merged against PS 2.4.x (not sure which version specifically). The many many changes between ps2.3 and 2.4 means a lot of refactoring and rework for Merrill so a full release will take time.  It’s pretty much a single person effort. 

The nightlies are there so you can try the latest and report issues. If you don’t know where they are then look under Actions. There’s a list of operating systems down the right hand side.  You can then download the latest build and extract and run as usual.

**Reply by DzzD** [reply]
> Define “new”. The latest nightly for windows was 6 days ago.

I was thinking of an official release, not a nightly build

>  Merrill so a full release will take time. It’s pretty much a single person effort

Yes, it must be a very tedious works, I even can't imagine what a nightmare it may be

>The nightlies are there so you can try the latest and report issues. If you don’t know where they are then look under Actions.

Yes I may switch to this branch and help if I can

**Comment by supermerill** [comment]
wanted to finish the mac signing stuff and some fixes before. Maybe i should have output a temporary one in the meantime...
You can expect one on thuesday.
